src/components/Shared/Types/MeetingVote.tsx

```diff
@@ -0,0 +1,14 @@
+import { Matter } from "./Matter";
+import { IndividualMeetingVote } from "./IndividualMeetingVote";
+import { MATTER_STATUS_DECISION } from "../../../constants/ProjectConstants";
+
+export type MeetingVote = {
+  /** the matter being voted upon */
+  matter: Matter;
+  /** the date the meeting occurred in a format consumable by a Date constructor*/
+  date: string;
```

Comment:
If date is going to consume a JavaScripte Date, then the type can be actually be changed to `Date` instead of `string`.

Reply:
I'll do that here and the other table as well.

Reply:
Ended up deleting it per To's comments.  I did change the other table.